components/webui/imports/api/ingestion/server/CompressionDbManager.js

```diff
@@ -20,53 +20,28 @@ class CompressionDbManager {
     }
 
     /**
-     * Retrieves the last `limit` number of jobs and the ones with the given
-     * job IDs.
+     * Retrieves compression jobs that are updated on or after a specific time.
      *
-     * @param {number} limit
-     * @param {number[]} jobIds
+     * @param {number} lastUpdateTimestamp in seconds
      * @return {Promise<object[]>} Job objects with fields with the names in
      * `COMPRESSION_JOBS_TABLE_COLUMN_NAMES`
      */
-    async getCompressionJobs (limit, jobIds) {
-        const queries = [];
+    async getCompressionJobs (lastUpdateTimestamp) {
+        const queryString = `
+            SELECT
+                id as _id,
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.STATUS},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.STATUS_MSG},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.START_TIME},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.UPDATE_TIME},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.DURATION},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.UNCOMPRESSED_SIZE},
+                ${COMPRESSION_JOBS_TABLE_COLUMN_NAMES.COMPRESSED_SIZE},
+                UNIX_TIMESTAMP(CURRENT_TIMESTAMP(3)) as retrieval_time
```

Comment:
It can be a personal preference, but i'll probably place it before `id as _id` to make it more obvious, since we renamed the field. i.e., that way we're grouping the renaming clauses together.